src/utils/common/AvgBlockTime.ts

```diff
@@ -0,0 +1,22 @@
+//
+// Calculates the average blocktime across a span of block objects
+//
+
+import { IBlock } from "types/NXTAPI";
+
+export function CalculateAvgBlocktime(blocks: Array<IBlock>) {
+  let prevBlock = blocks[0];
```

Comment:
you may want to try/catch this code since it assumes that the array being sent in always has a length > 0.

you can then decide if a default average of -1 or something is a good error response, or something else.

Reply:
fixed